### ISSUE TITLE
Auto-indentation for F# incorrectly indents

### ISSUE THREAD

**Issue by dsyme** [mono/monodevelop#3797]
Auto-indent behavious for F# when hitting "return" is behaving incorrectly.

### Repro:

If I have this

```fsharp
type Item =
   { Text: string; Description: string }
type Model =
   { Count : int
     Items : Item[] }
```
and my cursor is before the second “type” and I hit return

### Expected:

```fsharp
type Item =
   { Text: string; Description: string }

type Model =
   { Count : int
     Items : Item[] }
```
and my cursor is before the second “type” 

### Actual:

```fsharp
type Item =
   { Text: string; Description: string }

    type Model =
   { Count : int
     Items : Item[] }
```
note the 4 space indentation

### Workaround

To insert the blank line I have to go to the end of the long line and hit return.

> VS bug #563768

**Comment by cartermp** [comment]
Note that this will happen indefinitely. IIRC I filed a bug in bugzilla about this sometime last year, but I prefer that there is a GitHub bug here anyways.

**Comment by xamarin-release-manager** [comment]
Fixed in version 8.6.0.802 (master)

Pull Request #3856 merged by: Jason Imison
Author: mono
Commit: 542319619b9138f7699d7be74d5702736fc4736b (xamarin/monodevelop)

**Comment by xamarin-release-manager** [comment]
> [sync] [VS-14] Comment by **xamarinc**



Fixed in version 8.6.0.802 (master)



Pull Request #3856 merged by: Jason Imison  
 Author: mono  
 Commit: 542319619b9138f7699d7be74d5702736fc4736b (xamarin/monodevelop)
